botocore/utils.py

```diff
@@ -364,6 +366,39 @@ def is_global_accesspoint(context):
     return is_global
 
 
+# All code related to experimental plugins is considered a private interface and
```

Comment:
Are we able to group this with the other plugin code? Do we think this is more generally applicable outside of the plugin context?